src/commands/synthetics/utils/internal.ts

```diff
@@ -87,3 +88,97 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
 
   return undefined
 }
+
+type validTestOverridesValues = 'boolean' | 'number' | 'string' | 'string[]' | 'ExecutionRule'
```

Comment:
Unions are generally named singular, because they can only be one type at a time
```suggestion
type ValidTestOverrideValueTypeName = 'boolean' | 'number' | 'string' | 'string[]' | 'ExecutionRule'
```

Reply:
and you can put the `type OverridesValues = boolean | number | string | string[] | ExecutionRule` just below as well

\+ rename it to `TestOverrideValueType`